src/docs/faq.md

```diff
@@ -1,70 +1,99 @@
-* Please see [CoolStar's driver portal](https://coolstar.org/chromebook/driverlicense/login.html) for a step by step tutorial on purchasing drivers.
+### What OS should I use?
+* This depends on your use case. Only you can answer this question. <!-- Not ubuntu plz -->
 
-#### Can I get AVS or SOF drivers for free?
-* No. Only the Linux drivers are free.
-  
-#### How come audio isn't working on Ubuntu / Ubuntu forks?
-* Ubuntu and Ubuntu-based distros may have issues, try switching to another distro.
+### My internal keyboard doesn't work, can I use an external keyboard to get into developer mode?
+* No. The hotkeys will only work on the internal keyboard.
 
-#### What Linux distros are recommended?
-* See [this page](installing/installing-linux.md).
+### I just want to boot Linux from USB on my Chromebook, what do I need to do?
+* Check the Supported Devices page, to ensure your device has functional RW_LEGACY firmware available. If so, flash RWL (RW_LEGACY firmware) and use that.
 
-#### Why is there a license error in Device Manager after installing audio drivers?
-* You messed up the license setup. Please refer to [CoolStar's driver portal](https://coolstar.org/chromebook/driverlicense/login.html).
+### Can I disable power-on when I open my Chromebook?
+* No. In what situation would you open your chromebook without the intention of turning it on anyways.
 
-#### Can I hackintosh my Celeron/Pentium CPU?
+### My Chromebook powers back on when I turn it off. How can I fix this?
+* You don't. Specific Chromebooks have this bug which takes too much effort to debug. If you were looking to disable power-on when you open the lid because of this, they are completely unrelated.
+
+### I just installed the UEFI firmware, and now my device boots to a black screen that says 'shell' - what do I do?
+* You're in the EFI shell; the firmware boots there when it can't find a valid UEFI boot device, either externally (USB, SD) or internally. Just type `exit` to get back to the the UEFI settings menu.
+
+### Should I install 32 or 64-bit OS?
+* The UEFI Full ROM firmware only supports 64-bit OS, so be sure to use the 'x86_64' version of whatever OS you want to install. USB media created from 32-bit ISOs will simply fail to boot.
+
+### I'm running Legacy boot firmware now - can I switch to the UEFI firmware?
+* If you're using SeaBIOS, you will likely need to re-install. Otherwise it should work fine.
+
+### Help! I can't exit Developer Mode!
+* If you're trying to exit Developer Mode but get the error `WARNING: TONORM prohibited by GBB_FORCE_DEV_SWITCH_ON` or find the `Return to secure mode` button missing, you need to reset your GBB flags. You can do that by running [MrChromebox's firmware utility script](https://mrchromebox.tech/#fwscript) and selecting `3) Set Boot Options (GBB flags)`, followed by `5) Reset to factory default`. 
+  * Then exit developer mode by running `sudo crossystem disable_dev_request=1; reboot`
```

Comment:
Why crossystem and not the key on the dev mode screen?

Reply:
Idk I didnt write this

Reply:
> Why crossystem and not the key on the dev mode screen?

Because it's dev mode screen agnostic. Otherwise I'd have to write "if you have the old dev mode screen, press space to disable dev mode, and if you have the new dev mode screen, press the arrow keys to select "return to secure mode".

Reply:
@ethanaobrien do feel free to change it though if that's the majority opinion